Tests/DangerTests/GitHubTests.swift

```diff
@@ -2,6 +2,7 @@
 import DangerFixtures
 import XCTest
 
+// swiftlint:disable function_body_length
 final class GitHubTests: XCTestCase {
```

Comment:
<!--
  0 failure: 
  1 warning:  Type body should ...
  
  
  DangerID: danger-id-Danger;
  File: Tests/DangerTests/GitHubTests.swift;
  Line: 6;
-->

- :warning: Type body should span 200 lines or less excluding comments and whitespace: currently spans 273 lines (`type_body_length`)